sycl/test/basic_tests/parallel_for_type_check.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -fsycl -fsycl-device-only -D__SYCL_INTERNAL_API -O0 -c -emit-llvm -S -o - %s | FileCheck %s
+// RUN: %clangxx -fsycl -fsycl-device-only -D__SYCL_INTERNAL_API -c -emit-llvm -S -o - %s | FileCheck %s
```

Comment:
I understand the removal of `-O0` is because this test actually needs range rounding, so after disabling it by default for `O0`, we need to remove `O0` to reenable range rounding. Am I right?

Reply:
Yes, that is correct.